nav2_route/src/breadth_first_search.cpp

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) 2023 Joshua Wallace
```

Comment:
You can leave your copyright here for the adaptation, but put the original (C) line for Smac above it since this is derivative.